src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1539,8 +1605,15 @@ protected override void InvokeDefaultAction(string path)
                 if (isDirectory)
                 {
                     DirectoryInfo directory = new DirectoryInfo(path);
+                    InodeTracker tracker = null;
+
+                    if (recurse && Context.MyInvocation.BoundParameters.ContainsKey("FollowSymlink"))
```

Comment:
Modified the `-FollowSymlink` parameter to set `-Recurse`, and the `-Depth` parameter does.